src/main/java/com/odysseusinc/arachne/datanode/service/impl/DataSourceServiceImpl.java

```diff
@@ -319,7 +355,8 @@ public void updateHealthStatus(Long centralId, CommonHealthStatus status, String
     @Override
     public AutoDetectedFields autoDetectFields(DataSource dataSource) {
 
-        return new AutoDetectedFields(CommonModelType.CDM);
+        CommonDataSourceDTO dataSourceDTO = checkDataSource(dataSource);
```

Comment:
This wouldn't be called in Standalone mode

Reply:
yeah, this is right, but as I can see we call this only for NETWORK 
```
    public void createOnCentral(User owner, DataSource dataSource) {

        if (Objects.equals(NETWORK, dataNodeService.getDataNodeMode())) {
            AutoDetectedFields autoDetectedFields = autoDetectFields(dataSource);
```

Reply:
fixed